common-util/Contracts/addresses.jsx

```diff
@@ -110,6 +128,8 @@ export const safeMultiSend = {
   10200: ['0x40A2aCCbd92BCA938b02010E17A5b8929b49130D'],
   80001: ['0x40A2aCCbd92BCA938b02010E17A5b8929b49130D'],
   31337: ['0x9d4454B023096f34B160D6B654540c56A1F81688'],
+  42161: ['0x9d4454B023096f34B160D6B654540c56A1F81688'],
```

Comment:
```suggestion
  42161: ['0x40A2aCCbd92BCA938b02010E17A5b8929b49130D'],
```
as per https://github.com/safe-global/safe-deployments/blob/main/src/assets/v1.3.0/multi_send_call_only.json